runtime/web_platform_apis.md

```diff
@@ -25,9 +27,13 @@ Here is a list of web platform APIs Deno implements:
 - [`URLPattern`](https://developer.mozilla.org/en-US/docs/Web/API/URLPattern)
 - [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)
 - [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API)
+- [Web File API](https://developer.mozilla.org/en-US/docs/Web/API/File_API)
 - [Web Storage API](./web_storage_api.md)
 - [Web Workers API](https://developer.mozilla.org/en-US/docs/Web/API/Worker)
 - [`WebSocket`](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket)
+- [`WebGPU`](https://deno.land/api@v1.26.0)
+
+You can find the Deno reference for these APIs [here](https://deno.land/api@v1.26.0).
 
 ## `fetch` API
```

Comment:
@ry @kitsonk it's somewhat strange that we are spending so much time/detail on the `fetch` API here but not any of the other APIs/functions. Did the original author intend to eventually document the other functions/APIs as well and just didn't get around to it, or was there some other reason why fetch was singled out?

Reply:
It was mostly because it has some variance from the web standard we needed to document, where as the other ones didn't really have variance.  I think ideally now that we have integrated built in API documentation, we would want to move most of that to inline documentation.